crates/e2e/src/tests.rs

```diff
@@ -0,0 +1,15 @@
+#[test]
+#[should_panic(
+    expected = "Error establishing connection to a node at ws://0.0.0.0:9944. Make sure you run a node behind the given url!"
+)]
+fn fail_initialization_with_no_node() {
+    let client_init =
+        crate::Client::<crate::PolkadotConfig, ink_env::DefaultEnvironment>::new(
+            "ws://0.0.0.0:9944",
+        );
+    tokio::runtime::Builder::new_current_thread()
```

Comment:
You could avoid this boilderplate by using[ `#[tokio::test]`](https://docs.rs/tokio/0.2.3/tokio/attr.test.html). Needs the `macro` feature of tokio.